cmd/juju/commands/ssh.go

```diff
@@ -59,6 +59,11 @@ where it is available. This is done by inserting them between the target and
 a possible remote command. Refer to the ssh man page for an explanation 
 of those options.
 
+For k8s charms, the --container argument is used to identity a specific
+container in the pod. For charms which run the workload in a separate pod
```

Comment:
the default is the charm container, either way, isn't it?

Reply:
It is but in reality, the --container argument is only really relevant for v2 charms. I'll reword a little